src/private/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from "./menus";
 export * from "./interfaces";
-export * from "./privateAPIs";
\ No newline at end of file
+export * from "./privateAPIs";
+export * from "./conversations";
```

Comment:
please merge with latest change and export conversation API and interface explicit